explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -373,14 +378,17 @@ private object SpectroscopyModesTable:
                                range = s.wavelengthCoverage,
                                declination = dec
                              )
+
                          val sortedRows: List[SpectroscopyModeRow]    = rows.sortBy(_.enabledRow)
                          // Computes the mode overrides for the current parameters
                          val fixedModeRows: List[SpectroscopyModeRow] =
                            sortedRows
                              .map(
-                               _.withModeOverridesFor(s.wavelength,
-                                                      asterism.map(_.map(_.sourceProfile)),
-                                                      constraints.imageQuality
+                               _.withModeOverridesFor(
+                                 // Should this wv come from the grating, or is the obs wavelength?
```

Comment:
Do we know the answer to this?

Reply:
Nope, I'm trying to get help from the expert